stack/indexer/indexer-security-init.sh

```diff
@@ -42,11 +42,17 @@ fi
 
 # -----------------------------------------------------------------------------
 
+WAZUH_INDEXER_ROOT_CA=$(cat /etc/wazuh-indexer/opensearch.yml | grep http.pemtrustedcas | sed 's/.*: //')"
+WAZUH_INDEXER_ADMIN_PATH="${WAZUH_INDEXER_ROOT_CA%%/root-ca.*}"
```

Comment:
What if the key is not named root-ca??
you can use `dirname $file` to obtain the path to the directory

Reply:
Done https://github.com/wazuh/wazuh-packages/commit/59c81ff61ec2d872a466910d634b90ccef023e6a